Task-20/index.js

```diff
@@ -0,0 +1,60 @@
+class Human {
+    name;
+    age;
+    constructor (name, age) {
+        this.name = name;
+        this.age = age;
+    }
+    information () {
+        console. log(new Human(this.name, this.age))
+    }
+
+}
+
+class Auto {
+    brand;
+    model;
+    year;
+    number;
+    owner;
+
+    constructor (brand, model, year, number) {
+        this.brand = brand;
+        this.model = model;
+        this.year = year;
+        this.number = number;
+    }
+    assignOuner (name) {
+        if (name.age < 18) {
+            return alert (`Too young to be owner!`)
+        } {
+            this.owner = name
+        }
+    }
+    info() {
```

Comment:
```suggestion
    logInformation() {
```
 метод всеж таки логує а не повертає інфу
до того ж називайте методи консистентно (напркилад як вище)